res/values-hu/strings.xml

```diff
@@ -321,4 +321,20 @@
   <string name="pref_ctrl_string">CTRL ablak tartalma</string>
   <string name="pref_ctrl_string_summary">Karakterek sorozata a CTRL+? kombinációkhoz.</string>
   <string name="fullscreen">Teljes képernyő</string>
+  <string name="hostpref_wantsession_title">"Héj indítása"</string>
+  <string name="hostpref_wantsession_summary">"Kapcsoljuk ki, ha csak port átirányításra van szükségünk"</string>
```

Comment:
These strings already existed (see lines 121-122). Adding them here again broke the build.